src/python/pants/base/build_environment_test.py

```diff
@@ -1,8 +1,12 @@
 # Copyright 2020 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
No changes, other than merging the `tests/python` file into the one we already had in `src/python`.